composer.json

```diff
@@ -12,13 +12,11 @@
     ]
   },
   "require": {
-    "php": ">=7.1",
+    "php": "^7.1",
     "propel/propel": "~2.0@dev"
   },
   "require-dev": {
-    "phpunit/phpunit": ">=7.0",
-    "codeclimate/php-test-reporter": "dev-master",
-    "athens/standard": "*"
+    "phpunit/phpunit": "^7.0"
```

Comment:
Should be also including up to 9 if possible for latest PHP8+

Reply:
changed to "^7.0 || ^8.0 || ^9.0"